behave_django/management/commands/behave.py

```diff
@@ -45,18 +54,43 @@ def get_behave_options():
     return tuple(new_options)
 
 
+def get_behave_args():
+    """Remove command line arguments not accepted by behave."""
+    args = sys.argv[2:]
+    for new_arg in get_new_options():
+        args.remove(new_arg.get_opt_string())
+    return args
+
+
+def monkey_patch_behave(django_test_runner):
+    """Integrate behave_django in behave via before/after scenario hooks."""
+    behave_run_hook = ModelRunner.run_hook
+
+    def run_hook(self, name, context, *args):
+        if name == 'before_scenario':
+            django_test_runner.before_scenario(context)
+        behave_run_hook(self, name, context, *args)
+        if name == 'before_scenario':
+            django_test_runner.load_fixtures(context)
+        if name == 'after_scenario':
+            django_test_runner.after_scenario(context)
+
+    ModelRunner.run_hook = run_hook
+
+
 class Command(BaseCommand):
     help = 'Runs behave tests'
     option_list = BaseCommand.option_list + get_behave_options()
 
     def handle(self, *args, **options):
         # Configure django environment
-        django_test_runner = DiscoverRunner()
+        django_test_runner = get_runner(options)
         django_test_runner.setup_test_environment()
         old_config = django_test_runner.setup_databases()
```

Comment:
What was your reasoning behind making `ExistingDatabaseTestRunner` instead of just doing an if else statement here? Something like

```python
if not options['dry_run'] and not options['use_existing_database']:
    old_config = django_test_runner.setup_databases()
```